ts/parser.ts

```diff
@@ -116,3 +116,10 @@ export const parseLines = function({
 
   return data;
 };
+
+export default {
+  slugify,
+  isLabel,
+  isChordLine,
+  isLyricLine
+};
```

Comment:
Let’s not mix named exports (like `parseLines`) with default exports. Sticking with named exports seems cleaner to me: `export const slugify = function(…) { … };` (or `export function slugify(…) { … }`), and `import * as parser from "../parser"`.